tracing-subscriber/Cargo.toml

```diff
@@ -58,6 +59,7 @@ tracing-serde = { path = "../tracing-serde", version = "0.2", optional = true }
 
 # opt-in deps
 parking_lot = { version = "0.12.1", optional = true }
+chrono = { version = "0.4.26", optional = true }
```

Comment:
are there any default feature flags of `chrono` which we're not actually using? if so, it's probably a good idea to change this dependency to `default-features = false` and explicitly enable the feature flags that we actually need for the APIs we're using. this way, we don't inadvertently enable feature flags in downstream builds that we don't actually need.